.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,12 +1,19 @@
-**What kind of change does this PR introduce?** (check at least one)
+**What type of PR is this?**
+<!--
+Your PR title should be descriptive, and generally start with type that contains a subsystem name with `()` if necessary 
+and summary followed by a colon. format `chore/docs/feat/fix/refactor/style/test: summary`.
+Examples:
+* "docs: fix grammar error"
+* "feat(translator): add new feature"
+* "fix: fix xx bug"
+* "chore: change ci & build tools etc"
+-->
 
-- [ ] Bugfix
-- [ ] Feature
-- [ ] Refactor
-- [ ] Other, please describe:
+**What this PR does / why we need it**:
 
-**The PR fulfills these requirements:**
-
-- [ ] All tests are pasing: You can run `make test` to test it.
```

Comment:
IMO, the checklist is helpful to the contributors.

Reply:
> IMO, the checklist is helpful to the contributors.

Sure, imo but this part should check by github ci.

Reply:
> this part should check by github ci.

You are absolutely right. The new contributors just might not know how to run all unit tests. It could give them a friendly tip.

Reply:
hi @yuluo-yx  I think it's necessary to remind contributors to test locally before PR, or we should tell contributors that they can run `make test` for testing

Reply:
> hi @yuluo-yx I think it's necessary to remind contributors to test locally before PR, or we should tell contributors that they can run `make test` for testing

Maybe this part of the guidance should be located in the How to Contribute docs, and we can remind first time contributors in the pr tmpl to read the contribution docs and submit the pr later? cc @LinuxSuRen

Reply:
> > hi @yuluo-yx I think it's necessary to remind contributors to test locally before PR, or we should tell contributors that they can run `make test` for testing
> 
> Maybe this part of the guidance should be located in the How to [Contribute docs](https://github.com/LinuxSuRen/api-testing/blob/master/CONTRIBUTION.md), and we can remind first time contributors in the pr tmpl to read the contribution docs and submit the pr later? cc @LinuxSuRen

Reply:
I agree with you, maybe add to the tmpl to ask if the contributor has run tests.

Reply:
> I agree with you, maybe add to the tmpl to ask if the contributor has run tests.

tks, I've already submitted the changes, you can review again.

Reply:
Yes, we can reminder the contributor to read the contribution file. +1

Reply:
> Yes, we can reminder the contributor to read the contribution file. +1

yep, I open this [issue](https://github.com/LinuxSuRen/api-testing/issues/376), We can completed it the later. cc @Agility6